.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/pycqa/flake8
+    rev: 3.9.1
+    hooks:
+    -   id: flake8
+-   repo: https://github.com/pycqa/isort
+    rev: 5.8.0
+    hooks:
+    -   id: isort
+-   repo: https://github.com/psf/black
+    rev: 20.8b1
```

Comment:
Not a fan of having to maintain all these additional versions, they're bound to get out of sync with what's in riot/tox.

Is it possible to have precommit install a hook that just does:

```sh
riot run -s "isort|black|mypy"
```
?

Reply:
👍🏻 I am a fan of this approach

Reply:
@Kyle-Verhoog @brettlangdon I think technically this is possible if we define our own hooks and reference our own github repo from the `.pre-commit-config.yaml` file. I can try testing it out, but I'm not sure if this is the intended use case of `pre-commit`.

On another note, `pre-commit` does offer the `autoupdate` command which will automatically update all hooks to latest versions. It seems to me that every time we manually update `black|isort|mypy` in tox/riot, it wouldn't be much extra work to update the hooks as well?

Reply:
@Kyle-Verhoog @brettlangdon update: switched over from `pre-commit` to using `autohook` since it's super simple and supports using our riot commands (i.e. no need to maintain additional versions). Left out `isort` since it's not yet a riot command, but that can be relayed to a future PR 😄